### ISSUE TITLE
roachprod: cluster start fails with --secure and a UA config profile specified

### ISSUE THREAD

**Issue by ajstorm** [cockroachdb/cockroach#113310]
`roachprod start [cluster-name] --args "--config-profile=replication-source"   ` works correctly and starts a UA cluster.  If you specify the `--secure` flag at the end though, it fails to create the admin user:

```
19:02:54 cluster_synced.go:1829: storm-ua-bug-bash-2: checking certs.tar
19:02:55 cockroach.go:386: storm-ua-bug-bash-2: starting nodes
19:02:58 cockroach.go:935: storm-ua-bug-bash-2: initializing cluster
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
storm-ua-bug-bash-2: executing sql 1/1
19:09:09 cockroach.go:1000: not creating default admin user due to error: COMMAND_PROBLEM: exit status 1
19:09:09 cockroach.go:1015: storm-ua-bug-bash-2: setting cluster settings
```

Jira issue: CRDB-32864

**Comment by ajstorm** [comment]
@renatolabs, I'm assigning this one to you since you already have a fix for it in your latest PR.

**Comment by renatolabs** [comment]
Fixed by #113320.
